devel/icu/Portfile

```diff
@@ -64,8 +64,13 @@ set docdir      ${prefix}/share/doc/${name}
 
 configure.args              --disable-layoutex \
                             --disable-samples \
+                            --disable-icu-config \
                             --enable-static
 
+variant icu_config description {Enable icu-config} {
+    configure.args-replace  --disable-icu-config --enable-icu-config
+}
```

Comment:
This variant is not applicable to all of the subports, so it should be restricted to appearing only in those subports where it makes sense.

Reply:
@ryandesign can I ask you to share an example of port file where I can see how to do it?